modules/rte_mis_school/rte_mis_school.services.yml

```diff
@@ -21,3 +21,14 @@ services:
     arguments: ['@entity_type.manager', '@rte_mis_core.core_helper']
     tags:
       - { name: access_check, applies_to: _school_details_view_access_check }
+
+  rte_mis_school.school_print_css_alter_subscriber:
+    class: Drupal\rte_mis_school\EventSubscriber\SchoolEntityPrintCssAlterSubscriber
+    tags:
+      - { name: event_subscriber }
+
+  rte_mis_school.pdf_download_check:
+    class: Drupal\rte_mis_school\Access\SchoolPdfDownloadAccessCheck
+    arguments: ['@entity_type.manager', '@current_user']
+    tags:
+      - { name: access_check, applies_to: _pdf_download_check }
```

Comment:
Rename `_pdf_download_check` to `_school_pdf_download_access_check`

Reply:
ok.